### PR TITLE
doc/md: making the getting started section more consistent and clearer

### PR DESCRIPTION
**Pull request by zeevmoney**

### CODE REVIEW

File: doc/md/getting-started.md [hunk 1]
```diff
@@ -42,15 +42,13 @@ Go to the root directory of your project, and run:
 ```console
 go run entgo.io/ent/cmd/ent init User
 ```
+
 The command above will generate the schema for `User` under `<project>/ent/schema/` directory:
 
-```go
-// <project>/ent/schema/user.go
+```go title="<project>/ent/schema/user.go"
```
Comment (rotemtam) [inline doc/md/getting-started.md:48]: 💯

**Review by rotemtam — APPROVED** [review]
LGTM - thanks for this!

File: doc/md/getting-started.md [hunk 2]
```diff
@@ -612,4 +594,6 @@ Now when we have a graph with data, we can run a few queries on it:
     }
     ```
 
-The full example exists in [GitHub](https://github.com/ent/ent/tree/master/examples/start).
+## Full Example
+
+- The full example exists in [GitHub](https://github.com/ent/ent/tree/master/examples/start).
```
Comment (a8m) [inline doc/md/getting-started.md:599]:
```suggestion
The full example exists in [GitHub](https://github.com/ent/ent/tree/master/examples/start).
```

**Review by a8m — APPROVED** [review]
Looks good except for a minor comment. Thanks 🙏🏻

**Pull request merged**